Update enable-controlled-folders.md (#5010)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/enable-controlled-folders.md

> If the feature is set to **Audit mode** with any of those tools, the Windows Security app will show the state as **Off**.
> If you are protecting user profile data, we recommend that the user profile should be on the default Windows installation drive.

## Endpoint Manager

> 1. Sign in to the [Endpoint Manager](https://endpoint.microsoft.com) and open Endpoint Security.

Reviewer: ```suggestion
1. Sign in to the [Endpoint Manager](https://endpoint.microsoft.com) and open **Endpoint Security**.
```

2. Go to **Attack Surface Reduction** > **Policy**.

> 3. Select the Plaform , choose **Windows 10 and later** and Select the profile **Attack Surface Reduction rules** > Click **Create**

Reviewer: ```suggestion
3. Select **Platform**, choose **Windows 10 and later**, and select the profile **Attack Surface Reduction rules** > **Create**.
```

> 4.  **Name** the Policy and add **Description** > Click **Next**

Reviewer: ```suggestion
4.  Name the policy and add a description. Select **Next**.
```

> 5.  Scroll down to the bottom and Select **Enable Folder Protection** Drop down and Choose **Enable**

Reviewer: ```suggestion
5.  Scroll down to the bottom, select the **Enable Folder Protection** drop-down, and choose **Enable**.
```

> 6.  Select **List of additional folders that need to be protected** and **Add** the folders that needs to be protected.

Reviewer: ```suggestion
6.  Select **List of additional folders that need to be protected** and add the folders that need to be protected.
```

> 7.  Select **List of apps that have access to protected folders** and **Add** the apps that have access to protected folders.

Reviewer: ```suggestion
7.  Select **List of apps that have access to protected folders** and add the apps that have access to protected folders.
```

> 8.  Select **Exclude files and paths from attack surface reduction rules** and **Add** the Files and paths that needs to be excluded from Attack surface Reduction rules.

Reviewer: ```suggestion
8.  Select **Exclude files and paths from attack surface reduction rules** and add the files and paths that need to be excluded from attack surface reduction rules.
```

> 9.  Select the profile **Assignments**, assign to **All Users & All Devices**, and **Save**.

Reviewer: ```suggestion
9.  Select the profile **Assignments**, assign to **All Users & All Devices**, and select **Save**.
```

> 10.  Select **Next** to save each open blade and **Create**.

Reviewer: ```suggestion
10.  Select **Next** to save each open blade and then **Create**.
```

   > [!NOTE]

>    > Wilcard is supported for applications, but not for folders. Subfolders are not protected. Allowed apps will continue to trigger events until they are restarted.

Reviewer: ```suggestion
   > Wildcards are supported for applications, but not for folders. Subfolders are not protected. Allowed apps will continue to trigger events until they are restarted.
```
